### DISCUSSION TITLE
Shouldn't there be more comprehensive doucmentation for eza like there is for exa?

### DISCUSSION THREAD

**Discussion by YousufSSyed** [eza-community/eza#506] [Q&A, Howto, Help]
the exa documentation has pages for all of the features, and assuming that eza development is going to keep diverging then it should have its own pages too. I know there's https://eza.rocks buts there's only one page.

**Comment by cafkafk** [comment]
I think this ultimately comes down to a need for people to make that. PRs are definitely welcome, but I think personally my own time is best spend reviewing PRs, releasing eza, and slowly but surely making the integration tests fool proof, which will hopefully automate it enough to free up more time for stuff like this.
